pkg/controller/kieapp/defaults/defaults.go

```diff
@@ -139,35 +139,71 @@ func GetEnvironment(cr *api.KieApp, service kubernetes.PlatformService) (api.Env
 }
 
 func setProductLabels(cr *api.KieApp, env *api.Environment) {
-	setObjectLabels(cr, &env.Console)
-	setObjectLabels(cr, &env.Dashbuilder)
+	setObjectLabels(cr, &env.Console, getFormattedComponentName(cr, getConsoleName(cr)))
+	setObjectLabels(cr, &env.Dashbuilder, getFormattedComponentName(cr, constants.DashBuilder))
 	for index := range env.Servers {
-		setObjectLabels(cr, &env.Servers[index])
+		setObjectLabelsForServer(cr, &env.Servers[index])
 	}
-	setObjectLabels(cr, &env.SmartRouter)
-	setObjectLabels(cr, &env.ProcessMigration)
+	setObjectLabels(cr, &env.SmartRouter, getFormattedComponentName(cr, constants.Smartrouter))
+	setObjectLabels(cr, &env.ProcessMigration, getFormattedComponentName(cr, constants.ProcessMigration))
+}
+
+func getConsoleName(cr *api.KieApp) string {
+	env := string(cr.Spec.Environment)
+	consoleName := ""
+	if strings.Contains(env, constants.RhdmPrefix) {
+		consoleName = constants.RhdmDecisionCentral
+	} else if strings.Contains(env, constants.RhpamPrefix) {
+
+		if strings.Contains(env, constants.Production) {
+			consoleName = constants.RhpamBusinessCentralMon
+		} else {
+			consoleName = constants.RhpamBusinessCentral
+		}
+	}
+	return consoleName
```

Comment:
do we have unit test coverage for each of these possibilities?

Reply:
seems `constants.RhpamBusinessCentral` is being tested but not the other console possibilities?